src/js/msp/MSPHelper.js

```diff
@@ -2112,7 +2113,8 @@ MspHelper.prototype.crunch = function(code) {
                                                     buffer.push8(FC.ADVANCED_TUNING.ff_interpolate_sp)
                                                           .push8(FC.ADVANCED_TUNING.ff_smooth_factor)
                                                           .push8(FC.ADVANCED_TUNING.ff_boost)
-                                                          .push8(FC.ADVANCED_TUNING.vbat_sag_compensation);
+                                                          .push8(FC.ADVANCED_TUNING.vbat_sag_compensation)
```

Comment:
```suggestion
                                                          .push8(FC.ADVANCED_TUNING.vbat_sag_compensation);
```

Reply:
Hmm. I don't think so. Everything inside this ```if``` is just one statement and the semicolon should be at the end?

Reply:
Oops, don't know how I missed the next line